.pre-commit-config.yaml

```diff
@@ -64,6 +64,16 @@ repos:
       - id: black
         name: Format code
 
+  - repo: https://github.com/sirosen/check-jsonschema
+    rev: 0.4.0
+    hooks:
+      - id: check-github-workflows # Validate GitHub Workflows against the schema provided by Schemastore
+        files: ^\.github/workflows/
+        types: [ yaml ]
+      - id: check-azure-pipelines # Validate Azure Pipelines config against the schema provided by Microsoft
+        files: ^\.azure-pipelines/
+        types: [ yaml ]
```

Comment:
Is this really something that should be checked at every commit? That's the original purpose of pre-commit and many people (including me) use it that way. So I want it to be as lightweight as possible.

Reply:
well it shall be fast, on the other hand, you can say that you do not want to check python code when you do only edits in docs... :rabbit: but yes agree that it shall be light :)

Reply:
I'm fine with docs. But I don't think we need to verify our CI-Pipelines every time :)